bridge/opentracing/bridge.go

```diff
@@ -620,6 +620,25 @@ func (s fakeSpan) SpanContext() trace.SpanContext {
 	return s.sc
 }
 
+// adapt OpenTracing TextMapWriter to OpenTelemetry propagation.TextMapCarrier
+type textMapWriterAdapter struct {
+	t ot.TextMapWriter
+}
+
+func (a textMapWriterAdapter) Get(key string) string {
+	// not implemented
+	return ""
+}
+
+func (a textMapWriterAdapter) Set(key string, value string) {
+	a.t.Set(key, value)
+}
+
+func (a textMapWriterAdapter) Keys() []string {
+	// not implemented
```

Comment:
Are both the `textMap(Writer|Reader)Adapter` needed? It seems like they could be unified into a single type that would wrap both a OT propagation types.

Reply:
How were you thinking to do that?

Reply:
I would merge the two by including two fields for both types of OT carriers removing the unimplemented methods and replacing with the equivalent implemented ones. You would only add checks to the methods to see if the respective carrier field was set before preforming any actions.

Reply:
OK I gave it a go.